tools/apiview/parsers/cpp-api-parser/ApiViewProcessor/ApiViewProcessor.hpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "ApiViewDiagnostic.hpp"
+#include <map>
+#include <memory>
+#include <nlohmann/json.hpp>
```

Comment:
NIT: We don't have a guideline on this but should.  If applicable the header order should be alphabatized in groups that are

stdlib headers 
project headers 
external headers

Reply:
This is the header order that clang-format imposes. It puts local headers (`"foo"`) before public headers (`<foo>`).

I can't speak to why this is, but it's the case.

Reply:
That seems ok.  Why it splices nlohman in amongst the stl headers is the part I don't like.

Reply:
clang-format is simple: It sorts headers alphabetically.